db/db_impl/db_impl_open.cc

```diff
@@ -1239,7 +1211,7 @@ Status DBImpl::RecoverLogFiles(const std::vector<uint64_t>& wal_numbers,
       // write MANIFEST with update
       status = versions_->LogAndApply(cfds, cf_opts, edit_lists, &mutex_,
                                       directories_.GetDbDir(),
-                                      /*new_descriptor_log=*/true);
+                                      /*new_descriptor_log=*/false);
```

Comment:
Actually, https://github.com/facebook/rocksdb/pull/5856 requires that these version edits be written to a new MANIFEST. Thanks to the atomicity of `rename`, the CURRENT file either points to a new MANIFEST with all these version edits that bump the `log_number` for each column family, or  points to the old MANIFEST with non-of these version edits. We thought about using atomic group, but for compatibility, we chose not to when we landed https://github.com/facebook/rocksdb/pull/5856. For the purpose of this PR, I think it's better to keep the old behavior for correctness.

Reply:
I was not aware of this implication. Changed back to `true`.